website/project/model.py

```diff
@@ -2431,6 +2436,47 @@ def serialize(self):
             'is_registration': self.is_registration
         }
 
+    def _initiate_retraction(self, user, justification):
+        """Initiates the retraction process for a registration
+        :param user: User who initiated the retraction
+        :param justification: Justification, if given, for retraction
+        """
+
+        retraction = Retraction()
+        retraction.initiated_by = user
+        if justification:
+            retraction.justification = justification
+        retraction.initiation_date = datetime.datetime.utcnow()
+        retraction.state = 'pending'
+
+        # @todo(hrybacki) investigate why Node#admin_contributors isn't working
```

Comment:
`admin_contributors` seems to return admin contributors on parent nodes that have read-only access, not admins on this node.